Big upload includes significant content and scripting additions

bohdanhor opened the pull request:
index.html is finally work as it should have
javascript big chunk of updates: makes buttons work, interactivity, save to localstorage etc.
some updates to stylesheet as well.

bohdanhor committed:
try something new

bohdanhor committed:
update of styling + first interactivity

bohdanhor committed:
omg i need to stop doing it at night

bohdanhor committed:
upd cleaning up

bohdanhor committed:
update of input name and starter choice texts

bohdanhor committed:
Massive update in almost everything: add storage, style updates, content & scripts

bohdanhor committed:
Update: button works now! Small improvement in CSS. Heavy with JavaScript

bohdanhor committed:
Add working finishing button to index

Index.html works now.

ToedQueen208 commented on index.html:
```diff
@@ -3,11 +3,15 @@
+    <meta name="description" content="This site lets you choose your first partner Pokémon from the classic
+          Kanto trio: Bulbasaur, Charmander, or Squirtle. After choosing, you
+          can explore type effectiveness and see how your starter stacks up
+          against regional Gym Leaders." />
```
Really nice desciption with meta tags

ToedQueen208 commented on index.html:
```diff
@@ -18,83 +22,89 @@
+      <!-- Intro with name input -->
+      <section id="speech" name="speechSection" class="intro-container">
+        <section id="inputNameSection" class="speech-box">
+          <p id="intro0">This site lets you choose your first partner Pokémon from the classic
```
I like the use of the next button to carry on the journey for the user. 
Also the addition of using Javascript to show the users name is a nice personal touch

bohdanhor committed:
Cleaning redundant code in CSS and small adjusting in HTML (explained in descr.)

HTML:

Delete overuse of <br> in .starter-button, use padding instead. (As <br> were broken in Firefox browser).

Adjusted description, because apparently breaking line by pressing enter and not with <br> still breaks it in description? Saw it after sharing my code.

CSS:

Make starterSprite bigger.

Adjust .starter-button now that there is no <br>.

Delete redundant pieces of code.
